src/NHSDPublicBrowseAcceptanceTests.Tests/Steps/Authorization/Authorization.cs

```diff
@@ -0,0 +1,118 @@
+﻿using FluentAssertions;
+using NHSDPublicBrowseAcceptanceTests.Tests.Utils;
+using OpenQA.Selenium;
+using System;
+using TechTalk.SpecFlow;
+
+namespace NHSDPublicBrowseAcceptanceTests.Tests.Steps.Authorization
+{
+    [Binding]
+    public class Authorization
+    {
+        private readonly ScenarioContext _context;
+        private readonly UITest _test;
+
+        public Authorization(UITest test, ScenarioContext context)
+        {
+            _test = test;
+            _context = context;
+        }
+        
+        [Given(@"that a User is not logged in")]
+        public void GivenThatAUserIsNotLoggedIn()
+        {
+            var builder = new UriBuilder(_test.Driver.Url)
+            {
+                Host = "docker.for.win.localhost"
+            };
+            _test.Driver.Navigate().GoToUrl(builder.Uri.ToString());
```

Comment:
when the test first starts it lands on publicbrowse homepage.

Reply:
This is being removed altogether in the next commit